api/lib_test.go

```diff
@@ -506,13 +500,27 @@ func TestHackatomQuerier(t *testing.T) {
 	var qres types.QueryResponse
 	err = json.Unmarshal(data, &qres)
 	require.NoError(t, err)
-	require.Nil(t, qres.Err, "%v", qres.Err)
+	require.Equal(t, "", qres.Err)
 	var balances types.AllBalancesResponse
 	err = json.Unmarshal(qres.Ok, &balances)
 	require.Equal(t, balances.Amount, initBalance)
 }
 
 func TestCustomReflectQuerier(t *testing.T) {
+	type CapitalizedQuery struct {
+		Text string `json:"text"`
+	}
+
+	type QueryMsg struct {
+		Capitalized *CapitalizedQuery `json:"capitalized,omitempty"`
+		// There are more queries but we don't use them yet
+		// https://github.com/CosmWasm/cosmwasm/blob/v0.11.0-alpha3/contracts/reflect/src/msg.rs#L18-L28
```

Comment:
Does this make sense to test those here, or just in `wasmd`?

If we can call in and out, maybe what we really need to test there is how it ties with the custom wasmd handlers and this is enough for tests in go-cosmwasm